src/main/webapp/index.html

```diff
@@ -44,19 +53,21 @@ <h1>TheGlobetrotter</h1>
     <!-- Main section -->
     <div id="map-section">
       <div id="route-creation-section">
-        <div id="route-creation-name">Route name </div>
-        <div class="new-place" id="place-id-1">
-          <span class="fas fa-ellipsis-v"></span>Place 1</div>
-        <div class="new-place" id="place-id-2">
-          <span class="fas fa-ellipsis-v"></span>Place 2</div>
-        <div class="new-place" id="place-id-3">
-          <span class="fas fa-ellipsis-v"></span>Place 3</div>
-        <div class="new-place" id="place-id-4">
-          <span class="fas fa-ellipsis-v"></span>Place 4</div>
-        <div class="new-place" id="place-id-5">
-          <span class="fas fa-ellipsis-v"></span>Place 5</div>
-        <div class="new-place" id="place-id-6">
-          <span class="fas fa-ellipsis-v"></span>Place 6</div>
+        <div id="route-creation-name"> <p>Route name</p> </div>
+        <ul id="places-table">
+          <li class="new-place" id="place-id-1">
```

Comment:
Could you make this table bigger so that it would be more visible for the users?

Reply:
Increased it to 500px and change height to auto.

Reply:
Great, thanks